tardis/model/tests/test_density.py

```diff
@@ -10,24 +10,22 @@
 
 @pytest.fixture(scope="module", autouse=True)
 def to_hdf_buffer(hdf_file_path, simulation_verysimple):
-    simulation_verysimple.model.homologous_density.to_hdf(
-        hdf_file_path, overwrite=True
-    )
+    simulation_verysimple.model.to_hdf(hdf_file_path, overwrite=True)
 
 
 def test_hdf_density_0(hdf_file_path, simulation_verysimple):
-    actual = simulation_verysimple.model.homologous_density.density_0
+    actual = simulation_verysimple.model.density
     if hasattr(actual, "cgs"):
         actual = actual.cgs.value
-    path = os.path.join("homologous_density", "density_0")
+    path = os.path.join("model", "density")
     expected = pd.read_hdf(hdf_file_path, path)
     assert_almost_equal(actual, expected.values)
 
 
 def test_hdf_time_0(hdf_file_path, simulation_verysimple):
-    actual = simulation_verysimple.model.homologous_density.time_0
```

Comment:
Are we sure that `time_0` and `time_explosion` are equivalent?

Reply:
in this case they are because we shift everything to time_explosion. the `model` object only has things at the same time.